README.md

```diff
@@ -13,7 +13,7 @@ Add this to your `Cargo.toml`:
 
 ```toml
 [dependencies]
-num-rational = "0.1"
+num-rational = "0.2"
```

Comment:
Hmm, I hadn't updated this yet, since I'm not sure how long 0.2 until will be released, but I guess it's OK.